cli/src/pcluster/config/cluster_config.py

```diff
@@ -191,6 +191,13 @@
     SupportedVersionsValidator,
     UserNameValidator,
 )
+from pcluster.validators.slurm_settings_validator import (
+    SLURM_SETTINGS_COMPUTE_RESOURCE_DENY_LIST,
+    SLURM_SETTINGS_QUEUE_DENY_LIST,
```

Comment:
They are `SLURM_SETTINGS` meaning that they are defined by Slurm and we are `denying` their use.

Using here `CUSTOM` gives the impression that they are defined by the user, like a tag or something.

Reply:
I'm fine with that.

Just had an idea: can we have one single `SLURM_SETTINGS_DENY_LIST` dictionary so that we can easily extend the structure in the future?

So we will have something like:

```
SLURM_SETTINGS_DENY_LIST = {
    "Queue": [ ... ],
    "ComputeResource": [ ... ],
    "SlurmConf": [ ... ],
    ...
}
```

Please feel free to propose improvements.